policyengine_us/variables/gov/states/co/tax/income/additions/co_itemized_or_standard_deduction_addback_eligible.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class co_itemized_or_standard_deduction_addback_eligible(Variable):
```

Comment:
since eligible implies a choice
```suggestion
class co_itemized_or_standard_deduction_addback_required(Variable):
```